GitHubData/ResourcesModelJSONConverter.cs

```diff
@@ -0,0 +1,94 @@
+// <copyright file="ResourcesModelJSONConverter.cs" company="KinsonDigital">
+// Copyright (c) KinsonDigital. All rights reserved.
+// </copyright>
+
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using GitHubData.Models;
+
+namespace GitHubData;
+
+/// <summary>
+/// Converts a JSON to a <see cref="ResourcesModel"/> object.
```

Comment:
Refactor code docs to below:

```suggestion
/// Converts JSON data to a <see cref="ResourcesModel"/> object.
```